src/Authenticator/ChangePasswordHandler.php

```diff
@@ -0,0 +1,149 @@
+<?php
```

Comment:
Please add strict typing

Reply:
Unsure what this means. I've added type hinting, is there something else missing?

Reply:
https://www.php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration.strict

OK, added.